labs-ios-starter/Surveys/View/NotificationsCollectionViewCell.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright © 2020 Shawn James. All rights reserved.
+// NotificationsCollectionViewCell.swift
+
+import UIKit
+
+class NotificationsCollectionViewCell: UICollectionViewCell {
+    // MARK: - Outlets
+
+    @IBOutlet var messageLabel: UILabel! {
+        didSet {
+            updateViews()
+        }
+    }
+
+    @IBOutlet var deleteButtonBackground: UIVisualEffectView!
```

Comment:
You're gonna have to teach me all about this when we're out of the woods, sounds awesome